src/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express'); 
+const { swaggerUi, swaggerSpec } = require('../swagger');
+const errorHandler = require('./middlewares/errorHandlingMiddleware');
+const routes = require('./routes/routes');
+
+const app = express();
+const PORT = 3000; 
+ 
+app.use('/api', routes); 
+app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+
```

Comment:
It is also good practice to add notFoundMiddleware

Reply:
Thank you, I will add this changes in the next PR